compiler/ztests/sql/assignees.yaml

```diff
@@ -0,0 +1,26 @@
+
+script: |
+  super -z -I query.sql
+
+inputs:
+  - name: query.sql
+    data: |
+      FROM in.jsup
+      | UNNEST [...assignees,assignee]
+      | WHERE this IS NOT NULL
```

Comment:
This doesn't feel right since currently `FROM in.jsup | UNNEST [...assignees,assignee]` evaluates to:
```
{login:"joe"}
{login:"fred"}
error("missing")
{login:"sue"}
{login:"joe"}
{login:"sue"}
error("missing")
```
or is `error("missing")` equivalent to null for `this IS NOT NULL`?

Reply:
This came from a SQL query where there is no MISSING or error("missing") and the behavior works with our current `this!=null` semantics and is also consistent with SQL++ semantics, so I think we're ok.